### PR TITLE
Tweak emcc compiler

### PR DESCRIPTION
**Pull request by jeetiss**
hello @shuding 

I maintain `yoga-layout` as part of my work on the `react-pdf` package and it is painful)

so I wanna help with the new version of prebuilt yoga layout.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,19 +4,17 @@ all: clean dir wasm
...
 		-Iyoga \
 		-g0 \
-		-O2 \
+		-Os \
 		-flto \
 		--closure 1 \
-		-msimd128 \
 		-s WASM=1 \
 		-s WASM_ASYNC_COMPILATION=1 \
-		-s EXTRA_EXPORTED_RUNTIME_METHODS=['cwrap','setValue'] \
```
Comment (jeetiss) [inline Makefile:16]: `EXTRA_EXPORTED_RUNTIME_METHODS` is obsolete method and I can't find any information on why it is here so

@shuding why did you add this param, do I need to return it back?

File: Makefile [hunk 2]
```diff
@@ -28,11 +26,10 @@ wasm:
 		-s MALLOC="emmalloc" \
 		-s INCOMING_MODULE_JS_API=['instantiateWasm','locateFile']\
 		-s EXPORT_NAME="yoga" \
-		-s "DEFAULT_LIBRARY_FUNCS_TO_INCLUDE=['memcpy','memset','malloc','free','strlen']" \
```
Comment (jeetiss) [inline Makefile:31]: seems like this option comes from `asm.js` compilation

it doesn't work with the closure compiler without `EXPORTED_FUNCTIONS` param so I removed it.

docs: https://github.com/emscripten-core/emscripten/blob/main/src/settings.js#L995-L1004

File: Makefile (hunk 1 above)
Comment (shuding) [reply Makefile:16]: Sorry I can't recall - guess I saw it somewhere from emcc's issues. I guess we can remove it!

File: Makefile [hunk 3]
```diff
@@ -4,19 +4,16 @@ all: clean dir wasm
...
 		--closure 1 \
-		-msimd128 \
 		-s WASM=1 \
 		-s WASM_ASYNC_COMPILATION=1 \
-		-s EXTRA_EXPORTED_RUNTIME_METHODS=['cwrap','setValue'] \
-		-s DISABLE_EXCEPTION_CATCHING=1 \
-		-s AGGRESSIVE_VARIABLE_ELIMINATION=1 \
-		-s NO_EXIT_RUNTIME=1 \
+		-s USE_CLOSURE_COMPILER=1 \
+		-s USE_ES6_IMPORT_META=0 \
```
Comment (shuding) [inline Makefile:16]: Nice find!

**Comment by shuding** [comment]
Amazing PR, I tested it locally and it worked well - thank you!

**Pull request merged**